business_requirement/models/business.py

```diff
@@ -260,9 +260,13 @@ def name_search(self, name, args=None, operator='ilike', limit=100):
 
     @api.multi
     @api.returns('mail.message', lambda value: value.id)
-    def message_post(self, body='', subject=None, message_type='notification',
-                     subtype=None, parent_id=False, attachments=None,
-                     content_subtype='html', **kwargs):
+    def message_post(
```

Comment:
```suggestion
    def message_post(self, *args, **kwargs):
```

Reply:
Why? No, this should stick to the whole signature instead of putting things that way. If not, some of the arguments can be passed in kwargs although being positional.

Reply:
Oh true, change method signature it's a bad practice. Sorry for the noise. Thanks.